src/utils/getScorepads.js

```diff
@@ -0,0 +1,72 @@
+import sendRequest from '../utils/sendRequest'
+import getPictureSrc from '../utils/getPictureSrc';
+
+
+/**
+ * Creates a player as an HTML element.
+ *
+ * @param {Object} player the player to create the HTML element from
+ * @return {HTMLSpanElement} the HTML element representing the player
+ */
+function createPlayerElement(player) {
+    const playerElement = document.createElement('span');
+    playerElement.className = 'player';
+  
+    const picture = document.createElement('img');
+    picture.className = 'player-picture';
+    picture.src = getPictureSrc(player && player.picture);
+    playerElement.appendChild(picture);
+  
+    const name = player ? player.name : 'Undefined';
+    playerElement.appendChild(document.createTextNode(name));
+    return playerElement;
+  }
+  
+  function createScorepadElement(scorepad, gameName, onLoad) {
```

Comment:
In `doppelkopf.js` hatten wir hier noch Doku zu dieser Funktion, die muss beim Kopieren verloren gegangen sein :grimacing: 

Das hier fehlt:
```
/**
 * Creates a scorepad as an HTML list item.
 *
 * @param {Object} scorepad the scorepad to create the HTML element from
 * @return {HTMLLIElement} the HTML list item representing the scorepad
 */
```

Da fehlen dann auch noch die `@param` Tags für `gameName` und `onLoad`.